common/test/acceptance/pages/common/utils.py

```diff
@@ -100,3 +103,44 @@ def hover(browser, element):
     Hover over an element.
     """
     ActionChains(browser).move_to_element(element).perform()
+
+
+def enroll_user_track(browser, course_id, track):
+    """
+    Utility method to enroll a user in the audit or verified user track.  Creates and connects to the
+    necessary pages. Selects the track and handles payment for verified.
+    Supported tracks are 'verified' or 'audit'.
+    """
+    payment_and_verification_flow = PaymentAndVerificationFlow(browser, course_id)
+    fake_payment_page = FakePaymentPage(browser, course_id)
+    track_selection = TrackSelectionPage(browser, course_id)
+
+    # Select track and process payment
+    track_selection.visit()
+    track_selection.enroll(track)
+    if track == 'verified':
+        payment_and_verification_flow.proceed_to_payment()
+        fake_payment_page.submit_payment()
+
+
+def add_enrollment_course_modes(self, tracks):
+    """
+    Add the course modes to test Enrollment Track based content groups
+    """
+    audit_track = "Audit Track"
+    verified_track = "Verified Track"
+    for track in tracks:
```

Comment:
Hmmm, if this is going to be a general utility method, I don't like how "tracks" can only contain "Audit Track" or "Verified Track" (and there is no documentation that indicates what "tracks" can be. I felt squeamish about it before, but it was in a single test file.

I think that "tracks" should allow any of the mode slugs ("audit", "verified", "honor", etc.).